### DISCUSSION TITLE
Welcome to entretien-naturalisation-francaise Discussions!

### DISCUSSION THREAD

**Discussion by Rim-El-Ballouli** [Rim-El-Ballouli/entretien-naturalisation-francaise#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anakhoul** [comment]
Bonjour Rim,

STP j'ai besoin d'acces afin de préparer pour l'entretien pour ce Lundi.

Merci d'avance

**Reply by Rim-El-Ballouli** [reply]
Bonjour, 

l'accès est accordé 😉 Je vous souhaite bonne chance pour votre entretien. 
Courage et bonne journée

Rim
